search-commons/src/test/java/no/unit/nva/search2/ResourceClientTest.java

```diff
@@ -328,6 +328,32 @@ void searchWithUriReturnsOpenSearchAwsResponse(URI uri, int expectedCount) throw
             assertThat(pagedSearchResourceDto.totalHits(), is(equalTo(expectedCount)));
         }
 
+        @ParameterizedTest
+        @MethodSource("uriProvider")
+        void some(URI uri, int expectedCount) throws ApiGatewayException {
+
+            var query =
+                ResourceQuery.builder()
+                    .fromQueryParameters(queryToMapEntries(uri))
+                    .withRequiredParameters(FROM, SIZE)
+                    .withOpensearchUri(URI.create(container.getHttpHostAddress()))
+                    .build()
+                    .withRequiredStatus(PUBLISHED, PUBLISHED_METADATA);
+
+            var response = searchClient.doSearch(query);
+            var pagedSearchResourceDto = query.toPagedResponse(response);
+
+            assertNotNull(pagedSearchResourceDto);
+            if (expectedCount == 0) {
+                logger.debug(pagedSearchResourceDto.toJsonString());
+            } else {
+                logger.debug(pagedSearchResourceDto.toString());
+            }
+
+            assertThat(pagedSearchResourceDto.hits().size(), is(equalTo(expectedCount)));
+            assertThat(pagedSearchResourceDto.totalHits(), is(equalTo(expectedCount)));
+        }
+
```

Comment:
isn't this just a copy of another test with the same input?

Reply:
Midlertidig test :), fjerner denne.